runtime/kusama/src/lib.rs

```diff
@@ -1640,6 +1640,1251 @@ pub type Executive = frame_executive::Executive<
 /// The payload being signed in the transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+/// The nicks pallet was removed at block 569325 from the runtime, without any consideration of the
+/// fact that numerous accounts had reserved funds in this pallet. This migration is the outcome of
+/// an investigation that tries to refund all of the accounts that had a nick set for them prior to
+/// removal, if they still have the amount in their reserved balance. Otherwise, we ignore the
+/// refund.
+pub struct RefundNickPalletDeposit;
+
+impl RefundNickPalletDeposit {
+	fn execute(check: bool) -> Weight {
+		let accounts_and_deposits = vec![
+			(
+				[
+					116, 157, 220, 147, 166, 93, 254, 195, 175, 39, 204, 116, 120, 33, 44, 183,
+					212, 176, 192, 53, 127, 239, 53, 160, 22, 57, 102, 171, 83, 51, 183, 87,
+				],
```

Comment:
would like to have the account id written as a comment above these different hexs for easier audit

Reply:
that's a good idea actually. I will add them, please manually check a few of them? they are all auto generated, so if one is correct, they all should be fine.